runtime/gateway.go

```diff
@@ -486,14 +488,16 @@ func (gateway *Gateway) setupTChannel(config *StaticConfig) error {
 	serviceName := config.MustGetString("tchannel.serviceName")
 	processName := config.MustGetString("tchannel.processName")
 
-	subScope := gateway.AllHostScope.SubScope("tchannel")
```

Comment:
I liked the subscope idea.

Reply:
Let's keep the tchannel prefix to avoid collisons.

Reply:
Done.